README.md

```diff
@@ -166,6 +166,31 @@ def lambda_handler(event, context):
     }
 ```
 
+## Replacing the SSM client
+
+You can replace the SSM client for testing by setting the `SSMParameterStore.SESSION_FACTORY` and / or the `SSMParameterStore.CLIENT_FACTORY` class variables.
```

Comment:
Can you please update the documentation as well? :)

Reply:
Will do once all of the other comments below are fully resolved.